docs/source/contributing/developing.adoc

```diff
@@ -192,12 +192,19 @@ $go test -v <relative path of package> -run "Test<Regex pattern to match tests>"
 For more information about test options, run the `go test --help` command and review the documentation.
 
 [[integration-tests]]
-=== Running Integration Tests
+=== Integration Tests
 
-Integration tests utilize link:https://github.com/DATA-DOG/godog[Godog], which uses Gherkin (Cucumber) to define test cases.
-The test cases are defined in *_test/integration/features_*.
-By default, the tests are executed against the binary created by `make build`, that is *_$GOPATH/bin/minishift_*.
-To run the tests, run:
+Integration tests utilize link:https://github.com/DATA-DOG/godog[Godog], which uses Gherkin (Cucumber) to define sets of test cases, aka _features_ in Gherkin terminology.
+The features are located in *_test/integration/features_* folder.
+Features for Minishift are following 2 basic concepts:
+
+- *user stories* - follows a happy path of user, for example _basic.feature_ or _coolstore.feature_,
+- *feature and command coverage* - for example _proxy.feature_ or _cmd-version.feature_.
+
+===== Running integration tests
```

Comment:
* over-indentation - if you use === in the previous heading, then this heading should have ====
* Missing anchod ID above the heading, each heading should have an anchor ID that is unique and that matches the title text, so in this case it would be [[running-integration-tests]]